webpack/plugins/watch.js

```diff
@@ -1,17 +1,32 @@
+const path = require( 'path' );
 const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
+const WatchExternalFilesPlugin = require( 'webpack-watch-files-plugin' ).default;
```

Comment:
Curious about the `.default` usage here?

Reply:
It's the way this plugin exports, def a quirk. If they update at some point we may have to change this!